### PR TITLE
Second homework for check

### PR DESCRIPTION
**Pull request by nvkaip**

### CODE REVIEW

File: SecondHomeWorkApp.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public class SecondHomeWorkApp {
+    public static void main(String[] args) {
+        System.out.println(booleanExpression(false, true, true, false));
+        System.out.println(leapYearCount(200));
+        System.out.println(doubleExpression(0.1, 0.2, 0.3));
+        System.out.println(flipBit(122, 3));
+    }
+
+    public static boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
+        return (a == b && c == d && a != c) || (a == c && b == d && a != b) || (a == d && b == c && a != b);
```
Comment (nvkaip) [inline SecondHomeWorkApp.java]: I understand that variables names have to be more understandable, but this was given in the task and I'm a bit confused of naming them

File: SecondHomeWorkApp.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    public static int leapYearCount(int year) {
+        return year / 4 - year / 100 + year / 400;
+    }
+
+    public static boolean doubleExpression(double a, double b, double c) {
+        return Math.abs(a + b - c) <= (1E-4);
```
Comment (nvkaip) [inline SecondHomeWorkApp.java]: the same thing about naming variables

File: SecondHomeWorkApp.java (hunk 1 above)
Comment (Sofasmile) [reply SecondHomeWorkApp.java]: ok)

Comment (Sofasmile) [reply SecondHomeWorkApp.java]: you can make this task more simply

File: SecondHomeWorkApp.java (hunk 2 above)
Comment (Sofasmile) [reply SecondHomeWorkApp.java]: move 1E-4 into separate static final variable in the class

File: SecondHomeWorkApp.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package com.homework;
+
+public class SecondHomeWorkApp {
+
+    private static final double maxError = 1E-4;
```
Comment (Sofasmile) [inline SecondHomeWorkApp.java]: read about constant names
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

**Pull request merged**